admin/pre_reg/urls.py

```diff
@@ -3,10 +3,10 @@
 from . import views
 
 urlpatterns = [
-    url(r'^$', views.prereg, name='prereg'),
-    url(r'^drafts/(?P<draft_pk>[0-9a-z]+)/$', views.prereg_form, name='prereg_form'),
+    url(r'^(?:\?page=(?P<page_number>\d{1,5}))?$', views.prereg, name='prereg'),
```

Comment:
I'm pretty sure this isn't working. You just need to use `request.GET.get('page_number')`. I believe this is working because of the question mark at the end letting the `?page=00000` be optional. You can just switch this back to the previous line 6. `url(r'^$', views.prereg, name='prereg'),`